### ISSUE TITLE
identi.li

### ISSUE THREAD

**Issue by lain566** [jspenguin2017/uBlockProtector#472]
Can not see the download links

### Test link (required): 
http://www.identi.li/index.php?topic=480027
### Screenshot of the web page (required): 
![la_nueva_princesa_de_disney_ foto _ _la_rosquilla__2017-08-14_16-56-29](https://user-images.githubusercontent.com/27441315/29277452-f6201bd4-8111-11e7-81e6-7b448fdfdf3f.png)

### Screenshot of the console (press `F12` to bring up the console) (required): 
![developer_tools_-_httplarosquilla comla-nueva-_2017-08-14_16-56-54](https://user-images.githubusercontent.com/27441315/29277470-fceb2012-8111-11e7-8bb4-2c9f7ccdd088.png)

### Add everything else that you believe to be useful below (optional): 
 Scroll down and click on `Ver Links De Descarga` [Screenshot](https://user-images.githubusercontent.com/27441315/29279336-c7e8b180-8117-11e7-8b55-2c4fd197db5e.png)

A popup will open and this message appears
`Click on the advertising and wait 30 seconds will see links of downloads during 48 hours`

**Comment by jspenguin2017** [comment]
That button is not clickable.

**Comment by lain566** [comment]
Cannot open the popup?

**Comment by jspenguin2017** [comment]
Hum, it works after a refresh...

**Comment by lain566** [comment]
Should look like this
![relampago_en_la_oscuridad_ 2014 _-_identi_-_nightl_2017-08-14_18-18-21](https://user-images.githubusercontent.com/27441315/29280849-19c519e4-811d-11e7-80dc-6919f4953524.png)

**Comment by uBlock-user** [comment]
It wasn't clickable for me on the first load, disabling cosmetic filtering and re enabling made it clickable.

**Comment by uBlock-user** [comment]
http://www.identi.li/cacho/jquery_dt.js?v=7

**Comment by uBlock-user** [comment]
That eval runs the linkhidden code and runs the advert popup

**Comment by jspenguin2017** [comment]
Ehm... If you submit the first form on the page, then the link shows after 30 secs...

**Comment by jspenguin2017** [comment]
LOL That's a lot weaker than I expected...

**Comment by lain566** [comment]
@jspenguin2017  WOW Amazing! Thanks!!!

**Comment by uBlock-user** [comment]
Holy shit!!!, they just went into maintenance mode - 


![](https://i.gyazo.com/0b54a4942d92dd831648eeea80bfca8e.png)

**Comment by lain566** [comment]
@jspenguin2017 Another similar page, same message, can you fix it?
Scroll down and click on MEGA, Then click on ` Ver Enlaces De Descarga`
http://www.clasicotas.org/posts/hd1080p/3704/Rapido-Y-Furioso-1080p-Lat-Cast-Ing-5-1-2001-.html
![rapido_y_furioso_1080p_lat-cast-ing_5 1_ 2001 _-_c_2017-08-14_19-52-25](https://user-images.githubusercontent.com/27441315/29284367-b22bd058-812a-11e7-91e7-6026a278f1ff.png)
![adpop me_api_con_php_-_google_chrome_2017-08-14_20-04-33](https://user-images.githubusercontent.com/27441315/29284712-1a161182-812c-11e7-9004-21d5b332be8a.png)

**Comment by jspenguin2017** [comment]
It's actually quite different...

**Comment by lain566** [comment]
@jspenguin2017 Thanks! works fine.

I have another problem, just like here https://github.com/jspenguin2017/uBlockProtector/issues/389#issuecomment-321975390 I don't know if it's the same
Scroll down and click on `Paste` to see it
http://www.megawarez.org/ciguena-apuros-dvdrip-720p-1080p-latino/
![una_ciguena_en_apuros_2017_en_dvdrip _720p _1080p__2017-08-14_10-28-26](https://user-images.githubusercontent.com/27441315/29294216-8c833a0c-814e-11e7-8673-b2cf0af06af8.png)

**Comment by jspenguin2017** [comment]
This one is trivial and can be blocked by a cosmetic filter: 

```
megawarez.org##div.onp-sl-content:style(display: block !important)
megawarez.org##div.onp-sl-social-locker
```

Kind of weird, every social locker is different...

**Comment by lain566** [comment]
@jspenguin2017 I didn't think the solution was cosmetic filters.
Add this filter too, Easylist SPA list blocks popups from this domain, 
`||acortar.net$popup,badfilter`

**Comment by jspenguin2017** [comment]
Why should those popups be unblocked?

**Comment by lain566** [comment]
I explained wrong, 

Add this 
`||acortar.net$popup`
https://github.com/easylist/easylistspanish/commit/8cc49952a0cd352a33b1c20de9c24416bd65d09d

and tell me if can you enter the page acortar.net
here for example
https://github.com/jspenguin2017/uBlockProtector/issues/472#issuecomment-322270083
or
https://github.com/jspenguin2017/uBlockProtector/issues/472#issuecomment-322339934

**Comment by jspenguin2017** [comment]
Hum, interesting. But EasyList Spanish is not part of the standard setup, so it's up to you to add the `badfilter`.

**Comment by lain566** [comment]
@jspenguin2017 I said it because the person in charge of that list does not answer my messages. 
 Thanks anyway.

**Comment by lain566** [comment]
@jspenguin2017 Could you fix this?

These buttons are for displaying the links and password, but If you click on the buttons a popup opens and a countdown counter
http://mwpaste.com/?v=fIY
![una_ciguena_en_apuros_720p_vip__mwpaste com_-_pro_2017-08-18_22-37-53](https://user-images.githubusercontent.com/27441315/29489251-98c1c74e-851c-11e7-97cf-4b734ad8979f.png)

**Comment by jspenguin2017** [comment]
Yes, this one is pretty badly done. Easy to fix.

**Comment by lain566** [comment]
@jspenguin2017 Thanks :relaxed:

**Comment by lain566** [comment]
@jspenguin2017 This page has changed https://github.com/jspenguin2017/uBlockProtector/issues/472#issuecomment-322270083
Click on MEGA and you will be redirected to `acortalo.net` 
Before was called `acortar.net`
Maybe....same rule will work

**Comment by jspenguin2017** [comment]
Yep, seems to work.

**Comment by lain566** [comment]
@jspenguin2017 Thanks. 
Do you know how to kill the popup on the page?

**Comment by lain566** [comment]
@jspenguin2017 On identi's website, it still works, you can see the links.
But if you log in you can no longer see

User and pass, here
https://gist.github.com/lain566/e5ec4d6231c9c6538be081b136c00c90

Click on `Accede` and enter user and password
![relampago_en_la_oscuridad_ 2014 _-_identi_-_google_2017-08-23_19-07-46](https://user-images.githubusercontent.com/27441315/29630934-33d13d22-883e-11e7-9091-64acbe3975a9.png)

Test link
http://www.identi.li/index.php?topic=480027

**Comment by jspenguin2017** [comment]
Looks like the download links are no longer hard coded into DOM if you are logged in... 
How are you suppose to get the download links when you are logged in?

**Comment by lain566** [comment]
Click on` Descarga normal` and wait 15 seconds
![instrucciones_para_descargar_-_google_chrome_2017-08-24_01-13-35](https://user-images.githubusercontent.com/27441315/29642534-c22947e2-8869-11e7-99fc-02b49d572407.png)

**Comment by lain566** [comment]
This might be the script
http://www.identi.li/js/display15.js?v=11

**Comment by jspenguin2017** [comment]
Yea, they encrypted the download links this time...

**Comment by jspenguin2017** [comment]
But the decryption key is world readable, good job. Let me fix this.

**Comment by lain566** [comment]
Wow... I didn't think you'd make it :stuck_out_tongue_winking_eye:
Thanks

**Comment by lain566** [comment]
@jspenguin2017 Can you add this rule to prevent popups, only appears when you are not logged in
`a.cookie("BetterJsPop0", "1");`

**Comment by lain566** [comment]
@jspenguin2017 Is not showing the links. Only Google Drive link works.

Test link
http://www.identi.li/index.php?topic=481195
You'll have to log in.


![lolo](https://user-images.githubusercontent.com/27441315/29664867-cbbc0426-88d1-11e7-8853-be2b789f3d3f.png)


More test links
http://www.identi.li/index.php?topic=477952
http://www.identi.li/index.php?topic=481448

**Comment by jspenguin2017** [comment]
I don't have the password anymore...

**Comment by lain566** [comment]
@jspenguin2017 
```
lolo_
Alvaro_69
```

**Comment by jspenguin2017** [comment]
OK, thanks. Fixed.

**Comment by lain566** [comment]
@jspenguin2017 Same problem but with the JDownloader button.
This button has a feature to add links to JDownloader
You'll have to log in.

Test links
http://www.identi.li/index.php?topic=481675
http://www.identi.li/index.php?topic=481830
http://www.identi.li/index.php?topic=481806

![dragon_ball_super_capitulo_15_hdtv_latino_-_identi_2017-08-26_15-47-19](https://user-images.githubusercontent.com/27441315/29748005-7946eb2e-8b0c-11e7-8636-28f046810076.png)

**Comment by jspenguin2017** [comment]
Well... The real links are not hard coded into DOM this time. I see the link calls to `localhost`, and I couldn't find anything usable in the link.

**Comment by jspenguin2017** [comment]
I can get around watching ads, but I can't get around having to add link to Jdownloader.

**Comment by lain566** [comment]
@jspenguin2017 Thanks, that's all I need, and the system to add links to jdownloader is called [Click 'n' Load](http://jdownloader.org/knowledge/wiki/glossary/cnl2)
I think it is heavily encrypted.

**Comment by jspenguin2017** [comment]
From the docs, it looks like it is crackable, we have the ciphertext and we have the key. But we don't have the decryption algorithm. But that's a bit beyond bypassing the ads.

**Comment by jspenguin2017** [comment]
I thought Jdownloader is some adware from the website, but seems not, so if you don't like it, you can probably find an Userscript somewhere that can bypass it.

**Comment by lain566** [comment]
@jspenguin2017 Can you fix this? 
Scroll down and click on `Mostrar Enlaces De ....`

https://www.gratisjuegos.co/descarga/the-legend-of-zelda-oracle-of-seasons-espanol-mg-mf/

![the_legend_of_zelda_oracle_of_seasons_ espanol _ _2017-08-27_21-28-25](https://user-images.githubusercontent.com/27441315/29809046-dcf466a8-8c9a-11e7-9813-3a73f8170cd5.png)


More test links
https://www.gratisjuegos.co/descarga/ninjamurai-psp-4shared/
https://www.gratisjuegos.co/descarga/pack-zelda-64-roms-oot-majoras-mask-espanol-zippyshare/
https://www.gratisjuegos.co/descarga/the-legend-of-zelda-skyward-sword-wii-4shared-espanol-iso/

**Comment by lain566** [comment]
@jspenguin2017 Same filters as https://github.com/jspenguin2017/uBlockProtector/issues/472#issuecomment-322339934

http://www.descargatelocorp.com/mr-robot-temporada-2-completa-hd-espanol-latino/
Scroll down until you see it

![lolo](https://user-images.githubusercontent.com/27441315/29824594-3fb51c88-8cd2-11e7-8a16-b8246a9c873e.png)


And If you click on `Ver Enlaces`
A popup will open and this message appears:
`If you can not pass, disable the ad blockers`

![mr_robot_temporada_2_completa_hd_espanol_latino_-__2017-08-29_16-09-44](https://user-images.githubusercontent.com/27441315/29825708-817a9528-8cd5-11e7-9090-3869430357f7.png)

![download_-_google_chrome_2017-08-29_16-21-20](https://user-images.githubusercontent.com/27441315/29825919-22a2c6d2-8cd6-11e7-9dfc-8257fb640267.png)

**Comment by lain566** [comment]
@jspenguin2017 Can you fix this?
Scroll down until you see it

http://www.tipeo.net/posts/tvpeliculasyseries/8073/Castlevania-la-serie-netflix-latino-mega-.html
http://www.compucalitv.com/death-note-2017-hd-1080p-y-720p-latino/
http://www.goldescargas.com/kimi-no-na-wa-2016-brrip-1080p-sub-espanol/

![castlevania_la_serie_netflix_ latino _ mega _-_tip_2017-08-31_13-47-12](https://user-images.githubusercontent.com/27441315/29921766-2a0de360-8e53-11e7-8d1d-30ff1e45de16.png)
![death_note_ 2017 _hd_1080p_y_720p_latino_-_google__2017-08-31_13-43-11](https://user-images.githubusercontent.com/27441315/29921772-3367408c-8e53-11e7-9037-f979fbda3331.png)
![kimi_no_na_wa_2016_brrip_1080p_sub_espanol_ _golde_2017-08-31_13-43-43](https://user-images.githubusercontent.com/27441315/29921778-3b5ac5d4-8e53-11e7-98d3-49d1cfd8e723.png)

**Comment by jspenguin2017** [comment]
First one is pure style, third one doesn't have content hard coded, so have to boost timer.

**Comment by jspenguin2017** [comment]
Second one is pure style as well.

**Comment by lain566** [comment]
Thanks 👍

**Comment by lain566** [comment]
@jspenguin2017 They changed `Social Locker` https://github.com/jspenguin2017/uBlockProtector/issues/472#issuecomment-325683097 Can you fix it?
![mr_robot_temporada_2_completa_hd_espanol_latino_-__2017-09-08_21-11-05](https://user-images.githubusercontent.com/27441315/30227551-645e0846-94da-11e7-9404-bb273c0612b9.png)


Same rule as https://github.com/jspenguin2017/uBlockProtector/commit/5bf253bc95a211aa258724b17b5eb4d9af2e0b01 for:
http://www.intercambiosvirtualespro.org/iobit-driver-booster-pro-5/
http://www.tecmaxsoft.com/internet-download-manager-full-crack-serial-gratis/

![iobit_driver_booster_pro_5 0 2 1_rc_multilenguaje__2017-09-08_17-43-21](https://user-images.githubusercontent.com/27441315/30227919-cf9d3cb6-94db-11e7-9091-e76f4b7074e3.png)

![internet_download_manager_6 28_build_17_full_crack_2017-09-08_17-44-25](https://user-images.githubusercontent.com/27441315/30227932-db66f5e6-94db-11e7-8662-8801d3078f22.png)

**Comment by lain566** [comment]
@jspenguin2017 These filters are not working for me, I don't understand, it used to work.
```
intercambiosvirtualespro.org##div.onp-sl-content:style(display: block !important;)
intercambiosvirtualespro.org##div.onp-sl-social-locker
```

**Comment by lain566** [comment]
@jspenguin2017 Another website with `Social locker`, add this.
```
hackstore.net##div.onp-sl-content:style(display: block !important;)
hackstore.net##div.onp-sl-social-locker
```
http://hackstore.net/descargar-piratas-del-caribe-la-venganza-salazar-2017-blu-ray-rip-hd-latino/
<details><summary>Screenshot</summary>

![piratas_del_caribe_la_venganza_de_salazar_ 2017 __2017-09-27_17-36-50](https://user-images.githubusercontent.com/27441315/30922801-7f1eb6ec-a3aa-11e7-9927-fc4979ef76bc.png)
</details>

**Comment by lain566** [comment]
@jspenguin2017 How do I remove popup when I click on the button?

`https://www.gratispeliculas.xyz/descarga/the-crucifixion-2017-webrip-subtituladomg/`
<details><summary>Screenshot</summary>

![the_crucifixion_ 2017 _ webrip _ subtitulado mg _2017-10-09_21-47-38](https://user-images.githubusercontent.com/27441315/31355774-ddff0aa0-ad3b-11e7-9584-79fb8294af16.png)
</details>

**Comment by jspenguin2017** [comment]
I'll check it out once I finished implementing stage 1 of V12.

**Comment by jspenguin2017** [comment]
Like this? 
```
gratispeliculas.xyz##input[id^="contar"]
gratispeliculas.xyz##div[id^="hide"]:style(display: block !important;)
```

**Comment by lain566** [comment]
@jspenguin2017 Thanks, that's much better. Why didn't you do the same with this site?https://github.com/jspenguin2017/uBlockProtector/issues/472#issuecomment-325581533

This website is also related
`https://www.gratisprogramas.co/descargar/vide02brain-project-2016-avanzados-espanol-mega/`
<details><summary>Screenshot</summary>

![vide02brain_project_2016_avanzados_ espanol _ meg_2017-10-10_00-11-39](https://user-images.githubusercontent.com/27441315/31360658-b5474a28-ad4f-11e7-9bda-8229e06ac8f8.png)
</details>

**Comment by jspenguin2017** [comment]
That's a good question... I'll update it.

**Comment by lain566** [comment]
@jspenguin2017 It's not working, links aren't shown.
`http://www.identi.li/index.php?topic=491929`
<details><summary>Screenshot</summary>

![estreno-el_rico_y_lazaro_capitulo_12_hdligero_-_id_2017-11-15_12-59-06](https://user-images.githubusercontent.com/27441315/32835409-482ff36c-ca06-11e7-9cfa-607e6c2b937e.png)
</details>

------------------------------------------------------------------------------------------------------------

For example, in this link, works.
`http://www.identi.li/index.php?topic=491962`
<details><summary>Screenshot</summary>

![metal _5m_ mega _-_identi_-_google_chrome_2017-11-15_13-08-58](https://user-images.githubusercontent.com/27441315/32835480-8b02e4d8-ca06-11e7-8b19-7acea3675293.png)
</details>

**Comment by lain566** [comment]
By clicking on the link, I receive a 404 error Not found, the link is not dead.
<details><summary>Screenshot</summary>

![pcdj_dex_3_-_identi_-_google_chrome_2017-11-15_15-37-14](https://user-images.githubusercontent.com/27441315/32841773-874c553a-ca1b-11e7-88ea-491d4a8782a0.png)
</details>


Test links
`http://www.identi.li/index.php?topic=491951`
`http://www.identi.li/index.php?topic=491954`

**Comment by lain566** [comment]
It's broken, the button is not visible.
`http://www.identi.li/index.php?topic=281412`
<details>

![hajime_no_ippo__rising_3x11 _90mb _mega _848x480__2017-12-06_19-33-18](https://user-images.githubusercontent.com/27441315/33678635-9445f906-dabc-11e7-900d-0dc24ab2f34a.png)
</details>

**Comment by jspenguin2017** [comment]
Fixed.

**Comment by lain566** [comment]
thanks
